backend/src/modules/share_access/share_access.controller.ts

```diff
@@ -15,7 +15,7 @@ import { AccessService } from './share_access.service';
 // // Decorators
 // import { User } from '@app/decorators';
 
-@Controller('board-user')
+@Controller('share-access')
```

Comment:
Should create constanst controller for share-access.
Such as:
`
const SHARE_ACCESS_ROUTES = {
SHARE_ACCESS: 'share-access'
}
`